action.yml

```
@@ -1,11 +1,17 @@
 name: Gitolize
 description: Wrap a shell command with gitolize
 inputs:
+  ansi:
+    description: whether to keep ANSI escape sequences in the commit message
+    default: 'false'
   branch:
     description: which branch to pull from and push to
   command:
     description: shell command to be run by gitolize
     required: true
+  commit-sha:
```

What about a bit longer input name that might be more readable in places this action is used and the its description is not available?
```suggestion
  print-commit-sha:
```

---

Done in efb7c81e080558b9859da783bf75d4284667893d.